models/user.go

```diff
@@ -200,7 +200,7 @@ func (u *User) GetPhone() string {
 func (u *User) UpdateUserMetaData(tx *storage.Connection, updates map[string]interface{}) error {
 	if u.UserMetaData == nil {
 		u.UserMetaData = updates
-	} else if updates != nil {
+	} else {
```

Comment:
Is the `updates != nil` clause removed because it is redundant since iterating through a nil slice will produce a no-op?

Reply:
yup, the go linter actually pointed this out so i'm just removing it accordingly